Breeding-Insight/bi-web
# [BI-1129] View Germplasm List

# Description
**Story:** [BI-1129](https://breedinginsight.atlassian.net/jira/software/c/projects/BI/boards/1?modal=detail&selectedIssue=BI-1129)

- `GermplasmService.getAllInList()` created to fetch all germplasm in a given list
- `src/views/germplasm/GermplasmByList.vue` created for displaying list details
- `src/views/germplasm/GermplasmTable.vue`was renamed to `src/views/germplasm/AllGermplasm.vue`
- The germplasm table in `src/views/germplasm/AllGermplasm.vue` was factored out into a new component `src/components/germplasm/GermplasmTable.vue`
- 'src/views/germplasm/AllGermplasm' refactored to use `src/components/germplasm/GermplasmTable`
- the download button and modal was factored out of `src/components/germplasm/GermplasmListsTable.vue` into a new component, `src/components/germplasm/GermplasmDownloadButton`
- `src/components/germplasm/GermplasmListsTable,vue` refactored to use new download button component

# Dependencies
`feature/BI-1129` branch of bi-api

# Testing
- Import germplasm and navigate the germplasm lists table.
- Select the "Details" link on a row in the table
- Verify that the details displayed match the wireframe and spec


# Checklist:

- [x] I have run TAF: [3422049471](https://github.com/Breeding-Insight/taf/actions/runs/3422049471)
